Chapter12/recipe-02/fortran-example/account/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# specify where to place libraries
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
```

Comment:
Ditto as above for this one. Consider using `CMAKE_INSTALL_LIBDIR` from `GNUInstallDirs.cmake`